interface/patient_file/summary/create_portallogin.php

```diff
@@ -11,20 +11,25 @@
  * @author    Brady Miller <brady.g.miller@gmail.com>
  * @author    Tyler Wrenn <tyler@tylerwrenn.com>
  * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @author    Stephen Waite <stephen.waite@open-emr.org
  * @copyright Copyright (c) 2011 Z&H Consultancy Services Private Limited <sam@zhservices.com>
  * @copyright Copyright (c) 2018-2019 Brady Miller <brady.g.miller@gmail.com>
  * @copyright Copyright (c) 2020 Tyler Wrenn <tyler@tylerwrenn.com>
  * @copyright Copyright (c) 2022 Discover and Change, Inc <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Stephen Waite <stephen.waite@open-emr.org
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 require_once("../../globals.php");
 require_once('../../../library/amc.php');
 
-use OpenEMR\Common\Auth\AuthHash;
-use OpenEMR\Common\Csrf\CsrfUtils;
-use OpenEMR\Common\Twig\TwigContainer;
-use OpenEMR\Common\Utils\RandomGenUtils;
+use OpenEMR\Common\ {
+    Auth\AuthHash,
+    Csrf\CsrfUtils,
+    Logging\EventAuditLogger,
+    Twig\TwigContainer,
+    Utils\RandomGenUtils,
+};
```

Comment:
haven't seen this before. is there documentation on this (to ensure will work on older php versions). thoughts by others on this?

Reply:
think have put a few of these in the codebase in the recent past

https://www.php.net/manual/en/language.namespaces.importing.php#121785

Reply:
since that comment is many years old, then no worries on older php versions :)
fine by me

Reply:
This was an issue prior to PHP 7 but okay now